packages/react-renderer-demo/src/components/mdx/mdx-components.js

```diff
@@ -169,6 +170,7 @@ const MdxComponents = {
   inlineCode: ({ children }) => (
     <Root style={{ background: 'white', borderRadius: 3, fontFamily: 'courier, monospace', padding: '3px' }}>{children}</Root>
   ),
+  hr: () => <Divider variant={'middle'} style={{ marginBottom: 10, marginTop: 20 }} />,
```

Comment:
Can you use the `styled` HOC to create custom styling as it is done with other components? (The `Root` for the code component is a snowflake.)

Reply:
Sure, I'll update it

Reply:
I updated the code, please review~